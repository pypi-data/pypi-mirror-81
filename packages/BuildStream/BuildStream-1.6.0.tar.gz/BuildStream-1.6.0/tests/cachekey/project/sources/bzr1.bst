kind: import
sources:
- kind: bzr
  url: https://launchpad.net/bzr
  track: trunk
  ref: 6622
