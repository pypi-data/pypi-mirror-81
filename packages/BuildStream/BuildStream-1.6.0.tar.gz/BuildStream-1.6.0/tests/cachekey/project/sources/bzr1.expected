410a5606ccc28108db7bb9a57e017ede7435dbad27a8006e1c791b3253e9a56f